Resources/Locale/en-US/_Impstation/reagents/meta/consumable/drink/alchohol.ftl

```diff
@@ -37,7 +37,7 @@ reagent-desc-bloodier-mary = AH, AH, AHH!
 reagent-name-salvagerssoothe = salvager's soothe
 reagent-desc-salvagerssoothe = Come home safe, okay? We're counting on you.
 
-reagent-name-donttelldsheriff = don't tell D sherrif
+reagent-name-donttelldsheriff = don't tell D sheriff
```

Comment:
was this typo intentional? idk if it's a reference to something. although the key is spelt sheriff

Reply:
pretty sure it wasnt, searching the discord for "d sheriff" came up with the original suggestion https://discord.com/channels/716470896985899219/1277112023158886481/1278355436206161983